rskj-core/src/main/java/co/rsk/core/DifficultyCalculator.java

```diff
@@ -48,7 +48,7 @@ public BlockDifficulty calcDifficulty(BlockHeader header, BlockHeader parentHead
         return getBlockDifficulty(header, parentHeader, constants);
     }
 
-    private static BlockDifficulty getBlockDifficulty(
+    private BlockDifficulty getBlockDifficulty(
             BlockHeader curBlockHeader,
             BlockHeader parent,
             Constants constants) {
```

Comment:
Since the method is no longer static I think the `constants` parameter could be dropped